crt_portal/cts_forms/templates/forms/report_grouped_questions.html

```diff
@@ -1,30 +1,35 @@
 {% extends "forms/report_base.html" %}
 {% block form_questions %}
-  <h3 class="margin-bottom-1">
-    {{ step_question }}
-  </h3>
+  <div class="page-subtitle margin-top-4">{{ page_subtitle }}</div>
 
   {% if step_helptext %}
   <div class="margin-bottom-4"><em>{{ step_helptext }}</em></div>
   {% endif %}
 
   {% for question_group in wizard.form.question_groups %}
-    <h3 class="margin-bottom-1">
-      {{ question_group.group_name }}
-      {% if question_group.optional %}
-        <span class="question_group_optional_tag">(optional)</span>
+    <fieldset class="usa-fieldset margin-bottom-5">
+      <legend>
+        {{ question_group.group_name }}
+        {% if question_group.optional %}
+          <span class="question_group_optional_tag">(optional)</span>
+        {% endif %}
+      </legend>
+      {% if question_group.help_text %}
+        <div class="margin-bottom-3">
+          <em>{{ question_group.help_text }}</em>
+        </div>
+      {% else %}
+        <div class="margin-bottom-2"></div>
       {% endif %}
-    </h3>
-    <div class="margin-bottom-3"><em>{{ question_group.help_text }}</em></div>
 
-    {% for field in question_group %}
-      <fieldset class="usa-fieldset margin-bottom-1">
-        <legend class="margin-bottom-0">{{ field.label_tag }}</legend>
```

Comment:
This was an incorrect use of `<fieldset>` and `<legend>` — they should be use to encapsulate groups of questions instead of individual questions.